tests/yam/validate/validate_snapshot.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
```

Comment:
I would recommend to go with something like `validate_snapshots` in plural, it is quite generic, but we could add more later, otherwise in singular is confusing and should be called for clarity something more specific like`validate_after_install_snapashot_not_present.pm` which is too long, so change to `snapshots` could help.